src/components/ManageBar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { FaVoteYea, FaAngleDown } from 'react-icons/fa'
+
+function ManageBar() {
```

Comment:
let's try to make only named exports and avoid default exports, it may be a bit easier to make exports and imports in the future (i'll show you later why)

```suggestion
export function ManageBar() {
```